sirepo/template/radia.py

```diff
@@ -336,6 +337,16 @@ def python_source_for_model(data, model):
     return _generate_parameters_file(data, False, for_export=True)
 
 
+def validate_file(file_type, path):
+    if path.ext == ".stl":
+        mesh = _create_stl_trimesh(path)
+        if trimesh.convex.is_convex(mesh) == False:
+            return (f"not convex model")
```

Comment:
`return (f"not convex model: {path.basename}")`.

Reply:
The parens aren't needed too.

Reply:
validate_file is called automatically by server.py based on the code's template so I can remove file_path from the parameters but I'd likely have to tell the server to make a specific call to the function. Just removing it means this function doesn't get run.